### PR TITLE
Add lastOpenedAt key to communities

### PR DESCRIPTION
**Pull request by ibrkhalil**
Required for #17337

Adds the last opened at key to communities to know when was a community last opened (This is done currently for overview pages, Internal links not tested yet.)

status: ready

### CODE REVIEW

File: src/status_im/contexts/communities/overview/view.cljs [hunk 1]
```diff
@@ -374,14 +374,17 @@
       (rf/dispatch [:activity-center.notifications/dismiss-community-overview id]))
     [community-scroll-page community pending?]))
 
-(defn overview
+(defn internal-overview
   [id]
   (let [id                  (or id (rf/sub [:get-screen-params :community-overview]))
         customization-color (rf/sub [:profile/customization-color])]
+    (rn/use-effect #(rf/dispatch [:communities/update-last-opened-at id]) [id])
```
Comment (Parveshdhull) [inline src/status_im/contexts/communities/overview/view.cljs:381]: Hi @ibrkhalil, Thank you very much for your PR.
Maybe we can create another event, something like `:communities/navigate-to-community` (similar to `:chat/navigate-to-chat`) and handle this update there. wdyt?

File: src/status_im/contexts/communities/events.cljs [hunk 2]
```diff
@@ -169,6 +170,13 @@
                 {}
                 categories))}))
 
+(rf/reg-event-fx :communities/update-last-opened-at
+ (fn [_ [community-id]]
+   {:json-rpc/call [{:method     "wakuext_updateLastOpenedAt"
```
Comment (Parveshdhull) [inline src/status_im/contexts/communities/events.cljs]: Also, probably we should rename this end point. It somewhat gives idea that we are providing time at which community is opened.

Comment (Parveshdhull) [reply src/status_im/contexts/communities/events.cljs]: As Samuel mentioned in original issue, maybe something like `CommunityOpened`

File: src/status_im/contexts/communities/overview/view.cljs (hunk 1 above)
Comment (ibrkhalil) [reply src/status_im/contexts/communities/overview/view.cljs:381]: I think the only benefit for that would be better naming, Will do.

File: src/status_im/contexts/communities/events.cljs (hunk 2 above)
Comment (ibrkhalil) [reply src/status_im/contexts/communities/events.cljs]: What do you think about `wakuext_communityUpdateLastOpenedAt`

**Comment by pavloburykh** [comment]
Hey @ibrkhalil! What is the status of this PR? I see that it is not approved and remains in E2E column for couple of weeks already. 

Once you get PR approval, please rebase it (both mobile and status go branches) and add one of the labels (request manual qa or skip manual qa). 

In case manual qa is not required - we still need to rerun e2e tests after rebase. 

In case manual qa is required - please add testing notes in the PR description. 

Thank you.

**Comment by ibrkhalil** [comment]
Closed in favor of: https://github.com/status-im/status-mobile/pull/18439

**Pull request closed without merging**